modules/oe_editorial_unpublish/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenEuropa Editorial Unpublish
+
+This module provides an additional local task for moderated content that allows users with the
+appropriate permissions to unpublish it directly without having to go through
+the edit page first. 
+
+Unpublishing means moving the content to a state that has been configured in the workflow to change the status to 0.
+
+The "Unpublish" task is added automatically for content if the following conditions are met:
+
+* The bundle has content moderation enabled
+* The content's latest revision is published.
```

Comment:
The content has a revision that is published. -> This should be the generic rule.

Reply:
No, the correct one is:

`The content's default revision is published.`